Airports/EGNX/Positions_Mentor.txt

```diff
@@ -1,4 +1,4 @@
 EGNX_APP:East Midlands Radar:199.998:NXRM:X:EGNX:APP:-:-:4550:4571:N052.49.52.000:W001.19.41.000::
-EGNX_F_APP:East Midlands Director:120.120:NXFM:X:EGNX:APP:-:-:4550:4571:N052.49.52.000:W001.19.41.000::
+EGNX_F_APP:East Midlands Director:199.998:NXFM:X:EGNX:APP:-:-:4550:4571:N052.49.52.000:W001.19.41.000::
```

Comment:
Whilst this change is correct, it shouldn't be in this PR - issues and PRs should relate to one thing (in this case the SMR)

Reply:
Fixed and different issue and Pull completed